.github/workflows/ci.yml

```diff
@@ -52,6 +52,27 @@ jobs:
       - run: ctest --output-on-failure
         working-directory: build
 
+  windows_clang:
+    runs-on: windows-2022
+    strategy:
+      matrix:
+          compiler: [ clang, clang-cl ]
+          config: [ Release ]
```

Comment:
You aren't using the config parameter in the steps

Reply:
Oh right, I'll fix that.